lib/cronex/utils.rb

```diff
@@ -11,7 +11,14 @@ def include_any?(str, chars)
     end
 
     def number?(str)
-      Integer(str) rescue nil
+      Cronex::Utils.number(str) rescue nil
+    end
+
+    def number(str)
+      # Because fo following Integer() behavior
+      # https://www.ruby-forum.com/topic/62141
+      str.sub!(/^0+/, '')
+      Integer(str)
```

Comment:
not sure this is correct. It fails to treat a string with a single "0" as a valid integer.
```
>> def number(str); str.sub!(/^0+/, '') ; Integer(str); end
=> nil

# this does not work
>> num "0"
ArgumentError: invalid value for Integer(): ""
	from (irb):1:in `Integer'
	from (irb):1:in `number'
	from (irb):2
	from /Users/me/ruby/bin/irb:12:in `<main>'

# this works fine
>> number "09"
=> 9
```